src/components/localization/Keyboard.jsx

```diff
@@ -64,3 +64,37 @@ export const KeyboardSelector = ({ idPrefix }) => {
         </FormSelect>
     );
 };
+
+export const KeyboardGnome = () => {
+    const [compositorSelectedLayout, setCompositorSelectedLayout] = useState(null);
+
+    useEffect(() => {
+        const onFocus = () => getCompositorSelectedLayout().then(setCompositorSelectedLayout);
```

Comment:
This will not work. Why do we have it here?

Reply:
I think you re-viewed an outdated version?